open_discussions/settings.py

```diff
@@ -684,3 +684,13 @@ def get_all_config_keys():
 
 USE_X_FORWARDED_PORT = get_bool('USE_X_FORWARDED_PORT', False)
 USE_X_FORWARDED_HOST = get_bool('USE_X_FORWARDED_HOST', False)
+
+DJOSER = {
```

Comment:
Maybe add these to app.json (`PASSWORD_RESET_CONFIRM_URL`, `SET_PASSWORD_RETYPE`, `PASSWORD_RESET_CONFIRM_RETYPE`, `PASSWORD_RESET_SHOW_EMAIL_NOT_FOUND`)